src/content/docs/en/welcome.mdx

```diff
@@ -87,25 +87,25 @@ However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
-- Some distros choose to take stable release cycles to an extreme. These distros are designed for servers that will stay up for years by delaying updates for months or even a few years. PC users might miss out a lot by using these distros, including new features, bug fixes, security patches, etc. Without the proper knowledge and management, these systems could be even more insecure than Fedora/Ultramarine.
+- Packages in Fedora/Ultramarine are updated mid-release, but those updates **only include non-breaking changes**, mirroring the approach of releases such as Debian Stable by retaining ABI and API compatibility throughout the stable release cycle. Your computer has virtually zero chance of breaking mid-release due to an update.
+- Undeniably, Ultramarine and Fedora has much more frequent updates compared to other stable releases, and to some extent, they might be more unstable. However, Fedora/Ultramarine's release cycle makes it so that the operating system is still stable enough for normal use. Millions of Fedora devices across the world run fine without breaking, which demonstrate Fedora's high stability. Debian Stable's release cycle is also arguably too slow for desktop usage and can cause more hassles than necessary.
+- Some distros, primarily enterprise, choose to take stable release cycles to an extreme. These distros are designed for servers that will stay up for years by delaying updates for months or even a few years. Desktop users might miss out on important features and bug fixes with this release model. Without the proper knowledge and management, these systems could be even more insecure than Fedora/Ultramarine.
 
 ### Why Ultramarine (instead of other distros)
 
 - Performance
-  - Fedora (the distro Ultramarine is based on) has some of the best IO performance over other distros.
-  - BTRFS is the default file system, a relatively performant file system with features like snapshots. You can backup your files in less than a second (unlike other file systems like ext4).
-  - We include lightweight desktop environments such as Budgie (Flagship Edition) and XFCE. They look great, work well and are snappy.
+  - Fedora, the base of Ultramarine, has some of the best I/O performance over other distros.
+  - BTRFS is the default file system. It is a relatively performant and feature-complete file system with features like snapshots and improved performance on solid-state media. You can backup your files in less than a second (unlike other file systems like ext4).
+  - We include lightweight desktop environments such as Budgie (Flagship Edition) and XFCE. They look great and offer solid performance and usability.

```

Comment:
snappy makes more sense than performance here, we're talking about the experience not objective performance